src/ebfloeseg/load.py

```diff
@@ -22,7 +24,7 @@ class Satellite(str, Enum):
     aqua = "aqua"
 
 
-def get_width_height(bbox: str, scale: float):
+def get_width_height(bbox: BoundingBox | str, scale: float):
```

Comment:
I think it'd be good to have this function be a class method of `BoundingBox`. What do you think?

Reply:
I'm hesitant to include this function as part of the `BoundingBox`. I think keeping the `BoundingBox` as a pure dataclass which doesn't do anything helps me understand the code. 

Also, I'm really uncertain whether this function is so universal that we'd want to include it in the BoundingBox. My concern is that this function is very specific to the case that you have a bounding box and you want to get an integer number of pixels which will approximate its shape. What I'd expect the `BoundingBox.get_width_height` to do is to give me a float `width` and `height` if the values are floats, or integers if the values are all integers. The BoundingBox is much more – it might grow to include the `crs` which might give us another new meaning for `get_width_height`. So I think just keeping this `get_width_height` function closely associated with the `load` function is safest.

I'm going to mark it as private too.